site/docs.html

```diff
@@ -12,6 +12,19 @@
    }
    ul.doclist {
      font-size: 18px;
+     list-style-type: none;
+     padding-left: 1em;
+     padding-right: 1em;
+   }
+   li {
+     border-top: 1px solid #ddd;
+     padding: 0.12em;
```

Comment:
I think it looks good as-is, though I'd recommend
  padding: 0.2em 0;

... to add a smidge more padding (the parentheses in this font on Windows seem to descend pretty far). 0 horizontal padding to keep the dividers aligned with the text for a clean left edge at least. If you like it the way it is, keep it!

Reply:
yup agreed each of those suggestions is an improvement!